lib/google_drive/config.rb

```diff
@@ -0,0 +1,36 @@
+require 'json'
+
+module GoogleDrive
+  class Config
+    #:nodoc:
+    attr_accessor :client_id, :client_secret, :scope, :refresh_token, :config_path
+
+    Fields = %w(client_id client_secret scope refresh_token).freeze
```

Comment:
Can you call this FIELDS instead? Capital name is usually for class/module name, I guess.